source/Timer.cpp

```diff
@@ -1,315 +1,98 @@
 /*
- * Copyright (C) 2022 Sky UK
```

Comment:
We use lgpl license in rialto-gstreamer not apache

Reply:
changed